upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -172,15 +171,15 @@ func (e *AutoscalingGroup) Find(c *fi.Context) (*AutoscalingGroup, error) {
 			if len(resource) != 3 || resource[0] != "targetgroup" {
 				return nil, fmt.Errorf("error parsing target grup ARN resource: %q", parsed.Resource)
 			}
-			actualTGs = append(actualTGs, &TargetGroup{ARN: aws.String(*tg), Name: aws.String(resource[1])})
-		}
-		targetGroups, err := ReconcileTargetGroups(c.Cloud.(awsup.AWSCloud), actualTGs, e.TargetGroups)
-		if err != nil {
-			return nil, err
+			clusterName := fi.StringValue(g.AutoScalingGroupName)
+			if resource[1] != awsup.GetResourceName32(clusterName, "tcp") && resource[1] != awsup.GetResourceName32(clusterName, "tls") {
```

Comment:
I think if we're using this new GetResourceName32 function in the Find task then we should also use it in pkg/model/awsmodel

Reply:
Updated `GetELBName32` to use the new function.